package.json

```diff
@@ -42,36 +44,38 @@
     "webpack": "4.42.0"
   },
   "scripts": {
-    "start": "EXTEND_ESLINT=true react-scripts start",
-    "start:ci": "EXTEND_ESLINT=true npm-run-all build static",
-    "build": "EXTEND_ESLINT=true react-scripts build",
-    "test": "EXTEND_ESLINT=true react-scripts test --testMatch=[ \"**/?(*.)test.ts?(x)\" ]",
+    "start": "cross-env EXTEND_ESLINT=true react-scripts start",
+    "start:ci": "cross-env EXTEND_ESLINT=true run-s build static",
+    "build": "cross-env EXTEND_ESLINT=true react-scripts build",
+    "test": "cross-env EXTEND_ESLINT=true react-scripts test --testMatch=[ \"**/?(*.)test.ts?(x)\" ]",
     "new:component": "cremalab-scripts-web new component",
```

Comment:
We should probably add `cross-env` to the lib—though I suppose they are using cross-spawn... hmmm I wonder. It's probably worth adding the strategy matrix to our tests for `@cremalab/scripts` soon.

Reply:
Yeah I think that's the next place for me to make these changes, it's also a good time for me to get familiar with the process for `@cremalab/scripts`. 😅